test/Libraries/Microsoft.AspNetCore.HeaderParsing.Tests/ParserTests.cs

```diff
@@ -76,9 +76,7 @@ public void Cookkie_ReturnsParsedValue()
     {
         var sv = new StringValues("csrftoken=u32t4o3tb3gg43");
         Assert.True(CookieHeaderValueListParser.Instance.TryParse(sv, out var result, out var error));
-#pragma warning disable xUnit2013 // Do not use equality check to check for collection size.
         Assert.Equal(1, result.Count);
-#pragma warning restore xUnit2013 // Do not use equality check to check for collection size.
```

Comment:
revert all these

Reply:
If we are not using the new tooling though, I'm guessing these are not needed? Especially since they are just trying to silence an analyzer.